public/store/route/getters.ts

```diff
@@ -546,6 +546,13 @@ export const getters = {
     return task.includes(TaskTypes.REMOTE_SENSING);
   },
 
+  isGeoSpatial(state: Route): boolean {
+    // get tasks in route
+    const task = state.query.training as string;
+    // return if geospatial reside in the route hardcoded for now I dont believe there is an enum for it
+    return !!task && task.includes("__geo_coordinates");
```

Comment:
You should use a `TaskTypes` property instead of string, like you said in your comment.
Like on line 546

Reply:
I agree. However, this is checking the training variable not the task variable. The problem right now is geospatial is not supported as a task and I believe the server keeps removing it from the URL. Originally I went with that approach but until it is supported by the server we cannot use it.

Reply:
Talk to @phorne-uncharted, or @cdbethune, about adding it on the backend.
If they think it's not necessary, etc. that might help you refine this PR.